corehq/apps/app_manager/detail_screen.py

```diff
@@ -98,6 +100,10 @@ def __init__(self, app, module, detail, column, sort_element=None,
         self.sort_element = sort_element
         self.order = order
         self.id_strings = id_strings
+        self.parent_tab_nodeset = parent_tab_nodeset
+
+    def add_sort_node_for_nodeset_field(self):
```

Comment:
I used "add" because it decides if sort node is to be "added" for the column under nodeset. In column under nodeset we would want to add sort node if needed. So its not for the tab or the detail but for the column under it, hence referring to tab or nodeset detail in that name does not look right.
Can use `add_sort_node_for_nodeset_column` or `has_sort_node_for_nodeset_column` if that makes it better.